temporian/core/operators/simple_moving_average.py

```diff
@@ -39,9 +39,9 @@ def __init__(
         if sampling is not None:
             self.add_input("sampling", sampling)
         else:
-            sampling = data.sampling()
+            sampling = event.sampling()
 
-        self.add_input("data", data)
+        self.add_input("event", event)
```

Comment:
Nit: It is a bit hard to read the method because both the input and output are called "event" :)

Reply:
Good point - I made the change because we agreed that using `input_event` and `output_event` repeatedly was a bit too verbose. What other name could we use for these attributes?